src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx

```diff
@@ -117,7 +117,7 @@ export const BreachDetailsView = (props: Props) => {
               href={`https://${breach.Domain}`}
               eventData={{ link_id: breach.Domain }}
               target="_blank"
-              rel="noopener noreferrer"
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target `target="_blank"` automatically implies `noopener`, do we need to keep `noreferrer`?

Reply:
Good call - Amri told me once that they're no longer needed because browsers do it by default, but digging some more, for `noreferrer` it's actually that browsers set [a `strict-origin-when-cross-origin` referrer policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#strict-origin-when-cross-origin), which only sends the top-level domain. Sending nothing at all is probably even better. I'm just going to restore these `rel`s, because they don't do any harm either.

Reply:
6b0ea6965